features/beforeunload.yml

```
@@ -0,0 +1,11 @@
+name: Beforeunload
+description: "The `beforeunload` event is fired when the current window is about to be unloaded. Typically this is done to display a confirmation dialog that asks users to confirm if they really want to leave the page when they try to close or reload it, or navigate somewhere else."
```

```suggestion
description: "The `beforeunload` event is fired when the current window is about to be unloaded. Typically this is used to display a dialog to confirm if users really want to leave the page when there is unsaved data that would be lost."
```

A bit of tightening, but also mentioning data loss as it shouldn't be used just to prevent users from leaving.